Failed to build the image

I'm getting this error when trying to rebuild the image

```console
root@docker-builder-6:~/scaleway-camlistore# make release image_on_store
test armv7l = armv7l || make setup_binfmt
find patches -name '*~' -delete || true
docker build  -t camlistore:latest .
Sending build context to Docker daemon 199.7 kB
Sending build context to Docker daemon
Step 0 : FROM scaleway/ubuntu:vivid
 ---> 55676bb40402
Step 1 : MAINTAINER Mathieu Lonjaret <mathieu.lonjaret@gmail.com> (@lejatorn)
 ---> Using cache
 ---> baf298d5945d
Step 2 : RUN /usr/local/sbin/builder-enter
 ---> Using cache
 ---> 1f0acc5151b5
Step 3 : ENV DEBIAN_FRONTEND noninteractive
 ---> Using cache
 ---> 65706bc811f5
Step 4 : RUN apt-get update
 ---> Using cache
 ---> e2cef7f45e75
Step 5 : RUN apt-get -y upgrade
 ---> Using cache
 ---> 28fc1e419a16
Step 6 : RUN apt-get -y --no-install-recommends install curl gcc
 ---> Using cache
 ---> 143f8fbf908b
Step 7 : RUN apt-get -y --no-install-recommends install ca-certificates libc6-dev
 ---> Using cache
 ---> 42b4bdbfc518
Step 8 : WORKDIR /tmp
 ---> Using cache
 ---> 011d894730ae
Step 9 : RUN curl -O https://storage.googleapis.com/golang/go1.4.2.src.tar.gz
 ---> Using cache
 ---> 406ff08c5212
Step 10 : RUN echo '460caac03379f746c473814a65223397e9c9a2f6 go1.4.2.src.tar.gz' | sha1sum -c
 ---> Using cache
 ---> 88623aa2859a
Step 11 : RUN tar -C /usr/local -xzf go1.4.2.src.tar.gz
 ---> Using cache
 ---> b0beb7f52a5b
Step 12 : RUN mv /usr/local/go /usr/local/go1.4.2
 ---> Using cache
 ---> cac9497e8baa
Step 13 : WORKDIR /usr/local/go1.4.2/src
 ---> Using cache
 ---> 37169bda2124
Step 14 : ENV GOARCH arm
 ---> Using cache
 ---> ddc4f5d6bacd
Step 15 : ENV GOOS linux
 ---> Using cache
 ---> 56c6ba5e151a
Step 16 : RUN ./make.bash
 ---> Using cache
 ---> a1f58eb8c043
Step 17 : WORKDIR /tmp
 ---> Using cache
 ---> 404e9c225d93
Step 18 : RUN curl -O https://storage.googleapis.com/golang/go1.5.1.src.tar.gz
 ---> Using cache
 ---> 89aeee14e3f9
Step 19 : RUN echo '0df564746d105f4180c2b576a1553ebca9d9a124 go1.5.1.src.tar.gz' | sha1sum -c
 ---> Using cache
 ---> 421b51adc1c8
Step 20 : RUN tar -C /usr/local -xzf go1.5.1.src.tar.gz
 ---> Using cache
 ---> 3ebcc2d544bb
Step 21 : ENV GOROOT_BOOTSTRAP /usr/local/go1.4.2
 ---> Using cache
 ---> 93c00b893245
Step 22 : WORKDIR /usr/local/go/src
 ---> Using cache
 ---> 86aa3851d958
Step 23 : RUN ./make.bash
 ---> Using cache
 ---> 74b9fa969cef
Step 24 : RUN apt-get -y --no-install-recommends install git
 ---> Using cache
 ---> 063e166673e9
Step 25 : WORKDIR /tmp/src
 ---> Using cache
 ---> 6b8c7bb707f3
Step 26 : RUN git clone https://camlistore.googlesource.com/camlistore camlistore.org
 ---> Using cache
 ---> e336a2b7bc4d
Step 27 : WORKDIR /tmp/src/camlistore.org
 ---> Using cache
 ---> 9a251ad57d2d
Step 28 : ENV PATH $PATH:/usr/local/go/bin
 ---> Using cache
 ---> f6c94d61a53c
Step 29 : RUN go run make.go
 ---> Running in 7cefd66f37e3
camlistore.org/pkg/rollsum
camlistore.org/pkg/fileembed/genfileembed
camlistore.org/depcheck
camlistore.org/pkg/constants
camlistore.org/pkg/types
camlistore.org/vendor/golang.org/x/net/context
camlistore.org/pkg/hashutil
camlistore.org/pkg/legal
camlistore.org/pkg/netutil
camlistore.org/pkg/wkfs
camlistore.org/vendor/google.golang.org/cloud/internal
camlistore.org/pkg/context
camlistore.org/pkg/blob
camlistore.org/pkg/errorutil
camlistore.org/vendor/google.golang.org/cloud/compute/metadata
camlistore.org/pkg/buildinfo
camlistore.org/pkg/strutil
camlistore.org/pkg/osutil                                                                                                                                                                                                           [139/1259]
camlistore.org/pkg/syncutil
camlistore.org/pkg/camerrors
camlistore.org/pkg/env
camlistore.org/pkg/misc/gpgagent
camlistore.org/pkg/misc/pinentry
camlistore.org/pkg/jsonconfig
camlistore.org/third_party/code.google.com/p/go.crypto/openpgp/errors
camlistore.org/third_party/code.google.com/p/go.crypto/cast5
camlistore.org/pkg/httputil
camlistore.org/pkg/blobserver/protocol
camlistore.org/third_party/code.google.com/p/go.crypto/openpgp/armor
camlistore.org/third_party/code.google.com/p/go.crypto/openpgp/elgamal
camlistore.org/third_party/code.google.com/p/go.crypto/openpgp/s2k
camlistore.org/third_party/go/pkg/image/jpeg
camlistore.org/pkg/singleflight
camlistore.org/third_party/github.com/bradfitz/latlong
camlistore.org/pkg/blobserver
camlistore.org/third_party/github.com/rwcarlsen/goexif/tiff
camlistore.org/third_party/code.google.com/p/go.crypto/openpgp/packet
camlistore.org/pkg/auth
camlistore.org/third_party/github.com/rwcarlsen/goexif/exif
camlistore.org/pkg/readerutil
camlistore.org/pkg/geocode
camlistore.org/pkg/images/fastjpeg
camlistore.org/vendor/golang.org/x/image/math/f64
camlistore.org/third_party/github.com/nf/cr2
camlistore.org/vendor/golang.org/x/image/draw
camlistore.org/third_party/golang.org/x/image/tiff/lzw
camlistore.org/third_party/golang.org/x/image/tiff
camlistore.org/pkg/magic
camlistore.org/pkg/media
camlistore.org/pkg/schema/nodeattr
camlistore.org/third_party/code.google.com/p/leveldb-go/leveldb/db
camlistore.org/third_party/code.google.com/p/leveldb-go/leveldb/memdb
camlistore.org/pkg/types/camtypes
camlistore.org/pkg/sorted
camlistore.org/third_party/code.google.com/p/go.crypto/openpgp
camlistore.org/third_party/github.com/hjfreyer/taglib-go/taglib/id3
camlistore.org/third_party/github.com/gorilla/websocket
camlistore.org/third_party/github.com/hjfreyer/taglib-go/taglib
camlistore.org/pkg/jsonsign
camlistore.org/pkg/types/clientconfig
camlistore.org/pkg/legal/legalprint
camlistore.org/pkg/cmdmain
camlistore.org/pkg/blobserver/gethandler
camlistore.org/pkg/lru
camlistore.org/pkg/blobserver/memory
camlistore.org/pkg/schema
camlistore.org/pkg/fileembed
camlistore.org/pkg/misc/closure
camlistore.org/pkg/server/app
camlistore.org/server/camlistored/ui
camlistore.org/pkg/images/resize
camlistore.org/server/camlistored/ui/closure
camlistore.org/pkg/images
camlistore.org/third_party/code.google.com/p/rsc/gf256
camlistore.org/third_party/code.google.com/p/rsc/qr/coding
camlistore.org/third_party/code.google.com/p/xsrftoken
camlistore.org/third_party/fontawesome
camlistore.org/pkg/client/android
camlistore.org/pkg/index
camlistore.org/pkg/jsonsign/signhandler
camlistore.org/third_party/code.google.com/p/rsc/qr
camlistore.org/third_party/glitch
camlistore.org/third_party/less
camlistore.org/third_party/react
camlistore.org/third_party/github.com/garyburd/go-oauth/oauth
camlistore.org/pkg/importer/feed/atom
camlistore.org/pkg/importer/feed/rdf
camlistore.org/pkg/importer/feed/rss
camlistore.org/third_party/code.google.com/p/go-charset/charset
camlistore.org/third_party/code.google.com/p/go-charset/data
camlistore.org/third_party/code.google.com/p/go.net/html/atom
camlistore.org/third_party/code.google.com/p/goauth2/oauth
camlistore.org/third_party/code.google.com/p/go.net/html
camlistore.org/pkg/blobserver/local
camlistore.org/third_party/github.com/tgulacsi/picago
camlistore.org/pkg/blobserver/localdisk
camlistore.org/pkg/search
camlistore.org/internal/chanworker
camlistore.org/third_party/github.com/camlistore/lock
camlistore.org/pkg/cacher
camlistore.org/third_party/github.com/cznic/bufs
camlistore.org/third_party/github.com/cznic/fileutil
camlistore.org/third_party/github.com/cznic/mathutil
camlistore.org/third_party/github.com/cznic/sortutil
camlistore.org/third_party/github.com/cznic/zappy
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/util
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/comparer
camlistore.org/third_party/github.com/syndtr/gosnappy/snappy
camlistore.org/pkg/leak
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/cache
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/storage
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/filter
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/opt
camlistore.org/pkg/sorted/sqlkv
camlistore.org/pkg/sorted/sqlite
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/errors
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/iterator
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/journal
camlistore.org/pkg/blobserver/stats
camlistore.org/vendor/google.golang.org/api/googleapi/internal/uritemplates
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/memdb
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb/table
camlistore.org/vendor/google.golang.org/api/googleapi
camlistore.org/pkg/client
camlistore.org/pkg/server
camlistore.org/third_party/github.com/cznic/exp/lldb
camlistore.org/third_party/github.com/syndtr/goleveldb/leveldb
camlistore.org/cmd/camget
camlistore.org/pkg/sorted/leveldb
camlistore.org/third_party/github.com/cznic/kv
camlistore.org/vendor/google.golang.org/api/drive/v2
camlistore.org/pkg/importer
camlistore.org/pkg/kvutil
camlistore.org/pkg/sorted/kvfile
camlistore.org/pkg/blobserver/diskpacked
camlistore.org/pkg/blobserver/dir
camlistore.org/pkg/importer/dummy
camlistore.org/pkg/importer/feed
camlistore.org/pkg/importer/flickr
camlistore.org/pkg/importer/foursquare
camlistore.org/pkg/importer/picasa
camlistore.org/pkg/importer/pinboard
camlistore.org/pkg/importer/twitter
camlistore.org/cmd/camput
camlistore.org/pkg/blobserver/google/drive/service
camlistore.org/pkg/importer/allimporters
camlistore.org/dev/devcam
camlistore.org/vendor/golang.org/x/oauth2
camlistore.org/vendor/golang.org/x/oauth2/internal
camlistore.org/vendor/golang.org/x/oauth2/jws
camlistore.org/vendor/google.golang.org/api/storage/v1
camlistore.org/pkg/constants/google
camlistore.org/pkg/oauthutil
camlistore.org/pkg/blobserver/google/drive
camlistore.org/vendor/golang.org/x/oauth2/jwt
camlistore.org/vendor/golang.org/x/oauth2/google
camlistore.org/vendor/google.golang.org/api/logging/v1beta3
camlistore.org/pkg/googlestorage
camlistore.org/vendor/github.com/golang/protobuf/proto
camlistore.org/pkg/wkfs/gcs
camlistore.org/vendor/golang.org/x/net/internal/timeseries
camlistore.org/vendor/golang.org/x/net/trace
camlistore.org/vendor/google.golang.org/grpc/codes
camlistore.org/vendor/google.golang.org/grpc/credentials
camlistore.org/vendor/google.golang.org/grpc/grpclog
camlistore.org/vendor/google.golang.org/grpc/metadata
camlistore.org/vendor/github.com/bradfitz/http2/hpack
camlistore.org/vendor/google.golang.org/grpc/credentials/oauth
camlistore.org/vendor/github.com/bradfitz/http2
camlistore.org/pkg/blobserver/handlers
camlistore.org/pkg/types/serverconfig
camlistore.org/pkg/serverinit
camlistore.org/vendor/google.golang.org/grpc/transport
camlistore.org/third_party/labix.org/v2/mgo/bson
camlistore.org/third_party/github.com/go-sql-driver/mysql
camlistore.org/vendor/google.golang.org/grpc
camlistore.org/third_party/labix.org/v2/mgo
camlistore.org/vendor/google.golang.org/api/sqladmin/v1beta3
camlistore.org/vendor/google.golang.org/cloud/internal/opts
camlistore.org/third_party/github.com/lib/pq/oid
camlistore.org/third_party/github.com/lib/pq
camlistore.org/vendor/google.golang.org/cloud
camlistore.org/pkg/sorted/mysql
camlistore.org/vendor/google.golang.org/cloud/internal/transport
camlistore.org/pkg/sorted/postgres
camlistore.org/vendor/google.golang.org/api/compute/v1
camlistore.org/vendor/google.golang.org/cloud/storage
camlistore.org/vendor/google.golang.org/cloud/logging
camlistore.org/pkg/sorted/mongo
# camlistore.org/vendor/google.golang.org/cloud/storage
tmp/build-gopath-nosqlite/src/camlistore.org/vendor/google.golang.org/cloud/storage/storage.go:51: undefined: storage.DevstorageFullControlScope
tmp/build-gopath-nosqlite/src/camlistore.org/vendor/google.golang.org/cloud/storage/storage.go:55: undefined: storage.DevstorageReadOnlyScope
tmp/build-gopath-nosqlite/src/camlistore.org/vendor/google.golang.org/cloud/storage/storage.go:59: undefined: storage.DevstorageReadWriteScope
camlistore.org/pkg/conv
camlistore.org/pkg/pools
camlistore.org/third_party/go/pkg/archive/zip
camlistore.org/pkg/osutil/gce
camlistore.org/pkg/blobserver/cond
camlistore.org/cmd/camtool
camlistore.org/pkg/blobserver/encrypt
camlistore.org/pkg/blobserver/blobpacked
camlistore.org/pkg/blobserver/google/cloudstorage
camlistore.org/pkg/blobserver/mongo
camlistore.org/pkg/blobserver/proxycache
camlistore.org/pkg/blobserver/remote
camlistore.org/pkg/blobserver/replica
camlistore.org/pkg/fault
camlistore.org/pkg/misc/amazon/s3
camlistore.org/pkg/blobserver/shard
camlistore.org/pkg/throttle
camlistore.org/third_party/github.com/bradfitz/runsit/listen
camlistore.org/pkg/app
camlistore.org/pkg/webserver
camlistore.org/pkg/blobserver/s3
camlistore.org/pkg/publish
camlistore.org/third_party/bazil.org/fuse
camlistore.org/server/camlistored
camlistore.org/app/hello
camlistore.org/app/publisher
camlistore.org/third_party/bazil.org/fuse/fuseutil
camlistore.org/third_party/bazil.org/fuse/fs
camlistore.org/pkg/fs
camlistore.org/cmd/cammount
Error building main binaries: exit status 2
exit status 1
INFO[0255] The command [/bin/sh -c go run make.go] returned a non-zero code: 1
docker-rules.mk:194: recipe for target '.docker-container.built' failed
make: *** [.docker-container.built] Error 1
root@docker-builder-6:~/scaleway-camlistore#
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -38,6 +38,7 @@ RUN apt-get -y --no-install-recommends install git
 WORKDIR /tmp/src
 RUN git clone https://camlistore.googlesource.com/camlistore camlistore.org
 WORKDIR /tmp/src/camlistore.org
+RUN git reset --hard def28fc337ff85105b8b7dc6f0cda64af62b1615
 ENV PATH $PATH:/usr/local/go/bin
 RUN go run make.go
 RUN cp -a ./bin/* /usr/local/bin/

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 NAME =			camlistore
 VERSION =		latest
-VERSION_ALIASES =	tip
+VERSION_ALIASES =	def28fc337ff85105b8b7dc6f0cda64af62b1615
 TITLE =			Camlistore
 DESCRIPTION =		Camlistore with MySQL, pre-0.9
 SOURCE_URL =		https://github.com/scaleway-community/scaleway-camlistore

